packages/playground/src/pages/repo-settings-rules-page.tsx

```diff
@@ -0,0 +1,306 @@
+    console.log(data)
+    reset()
+  }
+  return (
+    <>
+      <form onSubmit={handleSubmit(onSubmit)}>
+        <FormFieldSet.Root>
+          <StackedList.Root className="border-none mb-8">
+            <StackedList.Item disableHover isHeader>
+              <StackedList.Field
+                title="Enable the rule"
+                description="Eget vestibulum eu lacinia ultrices id et morbi."
+              />
+              <StackedList.Field
+                label
+                secondary
+                title={
+                  <div
+                    className="flex gap-1.5 items-center justify-end cursor-pointer"
+                    onClick={() => setValue('toggleValue', !watch('toggleValue'))}>
+                    {' '}
+                    <Icon name={toggleValue ? 'toggle-active' : 'toggle-inactive'} size={30} />
+                  </div>
+                }
+                right
+              />
+            </StackedList.Item>
+          </StackedList.Root>
+          {/* NAME */}
+          <FormFieldSet.ControlGroup>
+            <FormFieldSet.Label htmlFor="name" required>
+              Name
+            </FormFieldSet.Label>
+            <Input id="name" {...register('name')} placeholder="Enter rule name" autoFocus />
+            {errors.name && (
+              <FormFieldSet.Message theme={MessageTheme.ERROR}>{errors.name.message?.toString()}</FormFieldSet.Message>
+            )}
+          </FormFieldSet.ControlGroup>
+          {/* DESCRIPTION */}
+          <FormFieldSet.ControlGroup>
+            <FormFieldSet.Label htmlFor="description" required>
+              Description
+            </FormFieldSet.Label>
+            <Textarea id="description" {...register('description')} placeholder="Enter a description of this rule..." />
+            {errors.description && (
+              <FormFieldSet.Message theme={MessageTheme.ERROR}>
+                {errors.description.message?.toString()}
+              </FormFieldSet.Message>
+            )}
+          </FormFieldSet.ControlGroup>
+
+          {/* PATTERNS */}
+          <FormFieldSet.ControlGroup>
+            <FormFieldSet.Label htmlFor="target-patterns" required>
+              Target Patterns
+            </FormFieldSet.Label>
+            <div className="flex gap-4">
+              <div className="flex-[2.5]">
+                <Input
+                  id="target-patterns"
+                  {...register('targetPatterns')}
+                  placeholder="Enter the target patterns"
+                  autoFocus
+                />
+                <Text size={2} as="p" color="tertiaryBackground" className="max-w-[100%]">
+                  Match branches using globstar patterns (e.g.”golden”, “feature-*”, “releases/**”)
+                </Text>{' '}
+              </div>
+              <div className="flex-[1]">
+                <Select value={includePatterns} onValueChange={value => handleSelectChange('includePatterns', value)}>
```

Comment:
this dropdown will only have `include/exclude` options. it should be a split button instead, similar to PR approve/reject button.